test/sink.test.js

```diff
@@ -42,3 +45,113 @@ test('.writer() - no value for "type" argument - should throw', () => {
         sink.writer();
     }).toThrowError('"type" is missing');
 });
+
+test.only('.writer() - happy path', async done => {
```

Comment:
don't think you meant to leave the `.only` in here!

Reply:
were there no node warnings about returning a promise and calling a callback when using `async done =>` ?

Reply:
thx. I dont think async and done usage is exclusive.